index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Personal Portfolio</title>
+</head>
+<body>
+    <nav id="navbar">
+        <ul>
+            <li>
+                <a href="#welcome-section">Welcome</a>
+            </li>
+            <li>
+                <a href="#projects">Check my projects</a>
+            </li>
+            <li> <a href="#contact">Contact me</a>
+            </li>
+        </ul>
+    </nav>
+    <section id="welcome-section">Welcome!
+        <H1> Welcome to my personal Portfolio</H1>
+    </section>
+    <section id="projects">
+        <p><h2> Here are some of my projects:</h2></p>
+        <div>
+            
+            <span id="project-header">
+            <p class="project-title"> Project #1</p>  
+              
+            </span>
+             <span id="project-body">
+               <img src='https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute.jpg' />
+             </span>
+              <span id="project-footer"><span>
+            <a href='' class="project-tile"></a>
+
+        </div>
+        <div>
+           
+             <span id="project-header">
+             <p class="project-title"> Project #2</p>
+              
+            </span>
+             <span id="project-body">
+               <img src='https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute.jpg' />
+             </span>
+              <span id="project-footer"><span>
+            <a href="https://github.com/jpcordone/microverse-basir-JP" class="project-tile"></a>
+
+        </div>
+           <div>
+           
+             <span id="project-header">
+            <p class="project-title"> Project #3</p>
+              
+            </span>
+             <span id="project-body">
+               <img src='https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute.jpg' />
+             </span>
+              <span id="project-footer"><span>
+            <a href="https://github.com/jpcordone/microverse-basir-JP" class="project-tile"></a>
+
+        </div>
+    </section>
+    <section id="contact">
+        <p1> You can contact me via:</p1>
+        <a id=profile-link href="https://github.com/jpcordone" target="_blank">GitHub</a>
+    </section>
+</body>
+<footer>
+
+</footer>

```

Comment:
Remove unused tags.

Reply:
Please always ensure your review comment is polite and not sounding like a command or order. 
* This is a polite suggestion: *"I think it would be better to remove unused code to make your codebase clean"*
* This reads as an order: *"Remove unused tags."*